worlds/sc2/item_descriptions.py

```diff
@@ -682,6 +682,11 @@ def _ability_desc(unit_name_plural: str, ability_name: str, ability_description:
     item_names.INFESTED_SIEGE_TANKS: "Mercenary infested Siege Tanks that may be called in from the Hatchery.",
     item_names.INFESTED_BANSHEES: "Mercenary infested Banshees that may be called in from the Hatchery.",
     item_names.OVERLORD_VENTRAL_SACS: "Overlords gain the ability to transport ground units.",
+    item_names.OVERLORD_GENERATE_CREEP: "Overlords gain the ability to generate creep while standing still.",
+    item_names.OVERLORD_SPAWN_CHANGELING: "Overlords can spawn Changeling unit.",
+    item_names.OVERLORD_CONTAMINATE: "The target structure will bew unable to train units or research upgrades for 30 seconds.",

```

Comment:
Typo: "bew".

Personally I'd rather have overseers than putting these upgrades on overlords. Abilities on overlords must necessarily be bad because they're on a unit that costs no supply and you'll have a bunch laying around for supply. I'd rather have better abilities linked to a gas unit so it's a decision to get them.

Reply:
Added Overseers that can spawn Tumors and Contaminate that also disables static defenses.